packages/modules/redis/src/redis-container.test.ts

```diff
@@ -0,0 +1,98 @@
+import { createClient } from "redis";
+import { RedisContainer, StartedRedisContainer } from "./redis-container";
+import * as os from "os";
+import * as path from "path";
+import * as fs from "fs";
+
+describe("RedisContainer", () => {
+  jest.setTimeout(240_000);
+
+  // startContainer {
+  it("should connect and execute set-get", async () => {
+    const container = await new RedisContainer().start();
+
+    const client = await connectTo(container);
+
+    await client.set("key", "val");
+    expect(await client.get("key")).toBe("val");
+
+    await client.disconnect();
+    await container.stop();
+  });
+  // }
+
+  it("should connect with password and execute set-get", async () => {
+    const container = await new RedisContainer().withPassword("test").start();
+
+    const client = await connectTo(container);
+
+    await client.set("key", "val");
+    expect(await client.get("key")).toBe("val");
+
+    await client.disconnect();
+    await container.stop();
+  });
```

Comment:
is it ok to not build the client using the password? 🤔

Reply:
The password is part of the Connection URL and will be used in the getConnectionUrl function which is used by connectTo.